src/go/rpk/pkg/cli/cmd/iotune/iotune.go

```diff
@@ -39,17 +40,31 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 			p := config.ParamsFromCommand(cmd)
 			cfg, err := p.Load(fs)
 			out.MaybeDie(err, "unable to load config: %v", err)
+
 			var evalDirectories []string
 			if len(directories) != 0 {
-				log.Infof("Overriding evaluation directories with '%v'",
+				fmt.Printf("Overriding evaluation directories with: %q\n",
 					directories)
 				evalDirectories = directories
 			} else {
 				evalDirectories = []string{cfg.Redpanda.Directory}
 			}
 
-			err = execIoTune(fs, evalDirectories, outputFile, duration, timeout)
-			out.MaybeDie(err, "error during IoTune execution: %v", err)
+			if exists, _ := afero.Exists(fs, outputFile); exists && !force {
+				out.Die("unable to run iotune: IO config file already exists in: %q; use --force if you want to overwrite its content", outputFile)
```

Comment:
The main existing flag we for forced runs is `--no-confirm` in both acl deletion and partition movement canceling. What do you think about switching this to a prompt if the file exists, and then using --no-confirm here?

Reply:
Done